htdocs/admin/faitdivers.php

```diff
@@ -0,0 +1,290 @@
+<?php
```

Comment:
faitdiverse  must be renamed into fediverse:  https://fr.wikipedia.org/wiki/Fediverse